Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/SANSFitShiftScaleTest.py

```diff
@@ -293,4 +293,5 @@ def test_that_error_is_transferred(self):
 
 
 if __name__ == "__main__":
+    FrameworkManager.Instance()
```

Comment:
In this test it uses the `AlgorithmManager` to create an algorithm object such as the `CreateWorkspace` algorithm. It seems the AlgorithmManager cannot find an algorithm called `CreateWorkspace` because the `simpleapi` module is no longer imported before the test is run. To ensure all the C++ and python algorithms are registered prior to running the test, I decided the best solution was to instantiate an instance of the `FrameworkManager`. This seems to be done commonly elsewhere in the code base e.g.

https://github.com/mantidproject/mantid/blob/84d4a52e028b8341b217c2d65868d2cc7e4a8d2a/Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/sans/SANSConvertToWavelengthAndRebinTest.py#L37